mobile/Trendwave/app/index.js

```diff
@@ -31,7 +31,8 @@ export default class Trendwave extends Component {
     super(props);
 
     this.state = {
-      popTweets: []
```

Comment:
Is it not safer to init. with empty arrays? (thinking of `Array#map` vs `null#map`)